tools/abel/model/entity_field.py

```diff
@@ -32,7 +32,8 @@
 from model.units import Units
 
 
-# TODO(b/229631364) Extend to telemetry field and metadata field classes
```

Comment:
these should have been removed by copybara. interesting they are leaving outside. can you please remove them

Reply:
done.